Makefile

```diff
@@ -391,6 +391,10 @@ lint: golangci-lint
 
 lint-fix: golangci-lint-fix
 
+check-permissions:
+	find ./ -executable -type f | grep -v "\.git" > ./executables.txt
+	diff <(sort ./executables.txt) <(sort ./contrib/executable_list.txt)
```

Comment:
Could we have some helpful comment to understand why we fail, in case of failure?

Reply:
okay, I've moved it to separate step in workflow + added comment there.
Example output when it fails:
https://github.com/cilium/cilium/actions/runs/15184968421/job/42703161351